etc/conf.templates/distributed/10-ingress.conf.template

```diff
@@ -277,3 +277,37 @@ ingress.kafka.metadata.maxsize = 900000
 // Identification of Ingress Metadata Update endpoint source
 //
 #ingress.metadata.update = 
+
+//
+// Default maximum age of datapoints pushed to Warp 10, in ms. Any timestamp older than
+// 'now' - this value will be rejected.
+// The maxpast value from the token will have precedence over this one
+//
+#ingress.maxpast.default =
+  
+//
+// Default maximum timestamp delta in the future for datapoints pushed to Warp 10, in ms.
+// Any timestamp more than this value past 'now' will be rejected
+// The maxfuture value from the token will have precedence over this one
+//
+#ingress.maxfuture.default = 
+  
+//
+// Absolute maximum age of datapoints pushed to Warp 10, in ms. Any timestamp older than
+// 'now' - this value will be rejected.
+// This value overrides both the default and token value for maxpast.   
+//
+#ingress.maxpast.override = 
```

Comment:
Is there a reason to prefer overriding instead of capping the value in a similar way stack limits are defined?
There would be three limits: default, token and max.

Reply:
The idea is to force a given value to ensure a specific behavior regardless of the limits that could be set in tokens.